ocs_ci/ocs/ui/storageclass.py

```diff
@@ -82,12 +92,20 @@ def delete_rbd_storage_class(self, sc_name):
 
         self.navigate_overview_page()
         self.navigate_storageclasses_page()
-        self.page_has_loaded()
+        self.page_has_loaded(retries=10)
         logger.info(f"sc_name is {sc_name}")
         self.do_click((f"{sc_name}", By.LINK_TEXT))
         self.do_click(self.sc_loc["action_inside_storageclass"])
         self.do_click(self.sc_loc["delete_inside_storageclass"])
         self.do_click(self.sc_loc["confirm_delete_inside_storageclass"])
         # wait for storageclass to be deleted
         time.sleep(2)
-        return not self.verify_storageclass_existence(sc_name)
```

Comment:
Can I ask why you removed this function?

Reply:
I haven't removed this function its on the else part of the condition on line 111